src/features/MentorPage/components/MentorList/MentorCard/List/ExpandButton.tsx

```diff
@@ -1,9 +1,10 @@
-import styled from 'styled-components';
-import { TextButton } from '@/components/Buttons';
-import { Mentor } from '@/features/MentorPage/mentorPageApi';
+import type { Mentor } from '@/features/MentorPage/mentorPageApi';
 
 import { useTranslation } from 'react-i18next';
 
+import styled from 'styled-components';
+import { TextButton } from '@/components/Buttons';
+
```

Comment:
I've put the styled-components on top with the other external dependencies.

But I think these are maybe not worth thinking about or changing too much when we don't have a consistent ordering system in place for imports.

Reply:
Yeah. I have grouped it with the components, because I kind of feel that its to do with the context of components. 
Same as variables and the palette. This is how I think it is readable in a ideal situation:

```javascript
// reactgroup
import {reactStuff, useState...}

// state-stuff
import {selector, actions}

// utility 
import {useSomething}
import {capitalize}

// component stuff
import {styled}
import {palette, variables }
import {Component}
```

But I have not been that consistent with every file. I fix these import things as I notice them. (or I try)
What do you think about this?

Reply:
I like to have one line space between the imports too, to make a gap between the groups

Reply:
Hmm if we use empty lines between import groups, we could enforce the sort-imports rule so that it forces the contents of  each group in alphabetical order. How about that?

Reply:
That sounds like a perfect solution